docs/guides/model-your-first-process.md

```diff
@@ -42,12 +48,14 @@ You can share read-only models with teammates and other stakeholders via a link.
 
 ## Collaborate on a process
 
-If you want to invite collaborators to work on your process together, you'll need to first make sure they have the proper permissions and roles.
+To invite collaborators to your process, ensure they have the proper permissions and roles:
 
-1. First, add your colleague to the Organization. Click your account name then **Organization Management**.
-2. Navigate to **Users > Add New User**. Add your colleague's email and assign them an appropriate role - **Developer** is likely a good default option here as the user will have full access to Console, Operate, and Tasklist without deletion privileges. See [all roles and permissions](/components/console/manage-organization/manage-users.md#roles-and-permissions) for additional details.
-3. Click **Add**. An email will be sent to the email you provided. Your colleague must hit **Join** to finish adding them to the organization.
+1. Select the **Open Organizations** icon in the top right of the navigation bar and select **Manage** next to your organization name.
+2. Navigate to **Users > Add new user**. Add your colleague's email and assign them an appropriate role - **Developer** is likely a good default option here as the user will have full access to Console, Operate, and Tasklist without deletion privileges. See [all roles and permissions](/components/console/manage-organization/manage-users.md#roles-and-permissions) for additional details.
+3. Select **Add**. An email will be sent to the email you provided. Your colleague must select **Join** to finish adding them to the organization.
 4. Now that they're added to the organization, you can add them to a project. Open **Modeler**, navigate to your project and open the **Collaborators** panel on the right side.
-5. Click **Add user** and find your colleague you added to your organization. Assign their role with the dropdown and click **Send invites**.
+5. Select **Add user** and find your colleague you added to your organization. Assign their role with the dropdown and select **Send invites**.
    ![add new user](./img/invite-collaborators.png)
-6. After your colleague clicks **Accept invitation**, they will have access to the project based on the role you assigned.
+6. After your colleague selects **Accept invitation**, they will have access to the project based on the role you assigned. To understand what permissions each role has, review [access rights and permissions](/components/modeler/web-modeler/collaboration.md#access-rights-and-permissions).
+
+As a next step, learn more about ensuring clarity, reusability, and governance within your organization in [using a shared project for organization-wide collaboration](/guides/use-shared-project-for-organization-wide-collaboration.md).
```

Comment:
I think we can add additional next steps, like orchestrating a process using a human task.